api/agent.go

```diff
@@ -136,6 +137,17 @@ type AgentMember struct {
 	DelegateCur uint8
 }
 
+// AgentMemberStatus corresponds the member status integers to the string status
+type AgentMemberStatus int
```

Comment:
It would be great to include a reference to the official source so that the reader understands this is a copy. Something like:

```
// AgentMemberStatus for a member of the cluster. The values correspond to those of
// github.com/hashicorp/serf/serf.MemberStatus.
```

Reply:
Ooh this is a good idea, thank you